circuit/src/execution-delivery/src/tests.rs

```diff
@@ -1245,3 +1252,57 @@ fn it_should_throw_with_empty_io_schedule() {
         expected
     );
 }
+
+#[test]
+fn test_authority_selection() {
+    let keystore = KeyStore::new();
+
+    // Insert Alice's keys
+    const SURI_ALICE: &str = "//Alice";
+    let key_pair_alice = sr25519::Pair::from_string(SURI_ALICE, None).expect("Generates key pair");
+    SyncCryptoStore::insert_unknown(
+        &keystore,
+        KEY_TYPE,
+        SURI_ALICE,
+        key_pair_alice.public().as_ref(),
+    )
+    .expect("Inserts unknown key");
+
+    // Insert Bob's keys
+    const SURI_BOB: &str = "//Bob";
+    let key_pair_bob = sr25519::Pair::from_string(SURI_BOB, None).expect("Generates key pair");
+    SyncCryptoStore::insert_unknown(
+        &keystore,
+        KEY_TYPE,
+        SURI_BOB,
+        key_pair_bob.public().as_ref(),
+    )
+    .expect("Inserts unknown key");
+
+    // Insert Charlie's keys
+    const SURI_CHARLIE: &str = "//Charlie";
+    let key_pair_charlie =
+        sr25519::Pair::from_string(SURI_CHARLIE, None).expect("Generates key pair");
+    SyncCryptoStore::insert_unknown(
+        &keystore,
+        KEY_TYPE,
+        SURI_CHARLIE,
+        key_pair_charlie.public().as_ref(),
+    )
+    .expect("Inserts unknown key");
+
+    // Alice's account
+    // let escrow: AccountId = hex_literal::hex!["d43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d"].into();
+
+    // Bob's account
+    let escrow: AccountId =
+        hex_literal::hex!["8eaf04151687736326c9fea17e25fc5287613693c912909cb226aa4794f26a48"]
+            .into();
+    let mut ext = TestExternalities::new_empty();
+    ext.register_extension(KeystoreExt(keystore.into()));
+    ext.execute_with(|| {
+        let submitter = ExecDelivery::select_authority(escrow.clone());
+
+        assert_eq!(submitter.is_ok(), true);
```

Comment:
You can use the assert! macro when the value is expected to be true.

Reply:
Added on the following commit